transport/http.go

```diff
@@ -144,6 +145,22 @@ func NewHTTPTransport() (*HTTPTransport, error) {
 		}
 	}
 
+	caCertPath := os.Getenv(envCACert)
+	if caCertPath != "" {
+		rootCAs, err := x509.SystemCertPool()
+		if err != nil || rootCAs == nil {
+			rootCAs = x509.NewCertPool()
+		}
```

Comment:
I think if you configure a custom CA cert, we should not be using the system cert pool at all.

```suggestion
		rootCAs := x509.NewCertPool()
```

Reply:
cool by me! will change.